core/src/test/scala/filodb.core/memstore/TimeSeriesPartitionSpec.scala

```diff
@@ -656,4 +657,23 @@ class TimeSeriesPartitionSpec extends MemFactoryCleanupTest with ScalaFutures {
     val data1 = part.timeRangeRows(AllChunkScan, Array(1)).map(_.getDouble(0)).toBuffer
     data1 shouldEqual (minData take 10)
   }
+
+  it("should return the current Chunk if the TimeRangeScan falls in current chunk") {
+    part = tracingPart(0, dataset1)
+    val data = singleSeriesReaders().take(11)
+    val minData = data.map(_.getDouble(1))
+    val initTS = data(0).getLong(0)
+    data.take(10).zipWithIndex.foreach { case (r, i) => part.ingest(0, r, ingestBlockHolder,
+      flushIntervalMillis = flushIntervalMillis,
+      acceptDuplicateSamples = acceptDuplicateSamples,
+      createChunkAtFlushBoundary = timeAlignedChunksEnabled)
+    }
+    val infos = part.infos(TimeRangeChunkScan(initTS, initTS + 100000000))
+    assert(!infos.isInstanceOf[FilteredChunkInfoIterator])
```

Comment:
Might be worth asserting that it's a `OneChunkInfo`

Reply:
I wanted to but it's a private class and I will have to change the visibility so left it there. I also thought of having  string search based assertion on the class name's toString but thought that wasn't the best way.